### ISSUE TITLE
to be translatable: toctree

### ISSUE THREAD

**Issue by shimizukawa** [sphinx-doc/sphinx#1734]
from #1235.
- toctree label

**Comment by pcav** [comment]
Also :description: would be useful, thanks.

**Comment by pcav** [comment]
For image, :target: is not translatable.

**Comment by mipapo** [comment]
```
.. toctree::
   :maxdepth: 1
   :caption: General
```

Would be great for caption

**Comment by tk0miya** [comment]
I made a PR #2880  to fix this.

@pcav what is `:description:`? The `toctree` directive does not have such an option.

BTW, I think modifying `:target:` option is not **translation**. So I won't change it.

**Comment by pcav** [comment]
I have something like:

```
.. meta::
   :description: likes and dislikes
   :keywords: 
```

**Comment by tk0miya** [comment]
Thanks for letting me know. I fixed it at #2883.
